fluent-syntax/src/serializer.ts

```diff
@@ -136,19 +136,29 @@ function serializeTerm(term: AST.Term): string {
 
 
 function serializeAttribute(attribute: AST.Attribute): string {
-  const value = indent(serializePattern(attribute.value));
+  const value = indentExceptFirstLine(serializePattern(attribute.value));
   return `\n    .${attribute.id.name} =${value}`;
 }
 
 
 function serializePattern(pattern: AST.Pattern): string {
   const content = pattern.elements.map(serializeElement).join("");
-  const startOnNewLine =
+  const maybeStartOnNewLine =
```

Comment:
I was puzzled, and then I realized that this var is not named right. I took this name face-value, and was very confused that you need to do two different checks, aka, this fix is not the fix I had in mind ;-)

How about `needsIndent` ?

Reply:
I moved this to a function called `shouldStartOnNewLine`, which I think is the right name this time.